.mci.yml

```diff
@@ -24,6 +24,8 @@ variables:
     ## cmake flag variables
     cmake_flags:
         ubuntu_cmake_flags: &ubuntu_cmake_flags -DCMAKE_CXX_FLAGS="-Wall -Wextra -Wno-attributes -Werror -Wno-error=missing-field-initializers"
+        ubuntu_boost_cmake_flags: &ubuntu_boost_cmake_flags -DBSONCXX_POLY_USE_BOOST=ON -DCMAKE_CXX_FLAGS="-Wall -Wextra -Wno-attributes -Werror -Wno-error=missing-field-initializers"
+        ubuntu_std_experimental_cmake_flags: &ubuntu_std_experimental_cmake_flags -DBSONCXX_POLY_USE_STD_EXPERIMENTAL=ON -DCMAKE_CXX_STANDARD=14 -DCMAKE_CXX_FLAGS="-Wall -Wextra -Wno-attributes -Werror -Wno-error=missing-field-initializers"
```

Comment:
And it looks like libc++ on that platform has experimental optional and string_view. I'm trying to build on a spawnhost right now.